.lintrunner.toml

```diff
@@ -216,6 +216,8 @@ include_patterns = [
     'torch/csrc/*.cpp',
     'torch/csrc/**/*.h',
     'torch/csrc/**/*.cpp',
```

Comment:
Hmm, quick question why doesn't
'torch/csrc/**/*.cpp' also cover  `'torch/csrc/distributed/autograd/*.cpp'`, smells like a bug where `**` isn't being expanded properly by lintrunner maybe?

Reply:
I think most of these issues are just because clang-tidy isn't new enough.

Reply:
> I think most of these issues are just because clang-tidy isn't new enough.

I had a PR to use clangtidy-18